flaml/model.py

```diff
@@ -1459,6 +1463,8 @@ def config2params(self, config: dict) -> dict:
             )
         if self._task not in CLASSIFICATION and "criterion" in config:
             params.pop("criterion")
+        if "random_state" not in params:
+            params["random_state"] = 12032022

```

Comment:
Seems odd to fix the random state here, might be surprising for users of the class. Maybe we can consider setting it for the test?

Reply:
The random state for other estimators is fixed. For random forest and extra trees it's not. I consider this a hidden bug that prevents reproducibility. I agree it could be surprising for users. I'll add it in documentation.

Reply:
Okay, makes sense :+1: